### PR TITLE
Change the `__iter__` implementation of the Config

### PR DESCRIPTION
**Pull request by nss10**
### Breaking Changes
* To be consistent with the behaviour of a dictionary, iterating through the config object would only yield a `key` instead of a `key,value` pair. Use `.items()` to replicate the `key,value` pair functionality.

### CODE REVIEW

**Review by paulineribeyre — DISMISSED** [review]

File: .github/workflows/pypi.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    tags:
+      - '*'
+jobs:
+  PyPIPoetryPublish:
+    name: PyPI Poetry Publish
+    uses: uc-cdis/.github/.github/workflows/python_package_index_publish.yaml@master
+    with:
+        PYTHON_VERSION: '3.9'
+        # the package was registered under someone's personal test-pypi account, so we
+        # can't push there
```
Comment (mfshao) [inline .github/workflows/pypi.yaml]: is this true for this repo? if so can we move it out to gen3?

Comment (nss10) [reply .github/workflows/pypi.yaml]: Thanks for the catch. I need to check on that. I got the pypi.yaml file from gen3utils and like a computer, my mind completely ignored the comments 😅

Comment (mfshao) [reply .github/workflows/pypi.yaml]: yeah lets verify that and I believe if this package is under gen3 on pypi, I also need to config the pypi creds for this repo in order for this workflow to work

Comment (nss10) [reply .github/workflows/pypi.yaml]: It appears there's no project named `gen3Config` on Test PyPI ([link here](https://test.pypi.org/search/?q=gen3config)). I believe we'll need to create one for the `uc-ctds-test` user. Is it something you can help me with @mfshao?

Comment (paulineribeyre) [reply .github/workflows/pypi.yaml]: It'll be created automatically, we just need to remove this comment from here

**Review by mfshao — APPROVED** [review]
i have added the creds to this repo

**Pull request merged**